PrestaShop/PrestaShop
# [-]BO: Fix employee drop down z-index

## Description

Fix the drop down of the employee on the upper-right of the screen on the catalog page.


## Steps to Test this Fix


1. Go to the catalog page,
2. Clic on the employee picture on the upper-right of the screen,
3. Check if the drop down is over the page header, not under.
